### PR TITLE
[feature fix]Feature/allow special characters in contributor search

### PR DESCRIPTION
**Pull request by brandonPurvis**
## Purpose
Currently, when adding a contributor to a project, searching for a name with special characters such as accents yields no results, and logs an error in the console. This PR resolves this issue causing searches to return the relevant users.

## Changes
Given a query with special characters, normalizes each word in that query converting all characters to ascii prior to passing the query to elasticsearch.

Closes Issue https://github.com/CenterForOpenScience/osf.io/issues/2460(https://github.com/CenterForOpenScience/osf.io/issues/2460)

### CODE REVIEW

**Comment by lbanner** [comment]
:+1: Nice

**Comment by GaryKriebel** [comment]
@brandonPurvis like with https://github.com/CenterForOpenScience/osf.io/pull/2858, in future PRs, please have the title describe the PR .... it is good to keep the issue number(s) the PR fixes in the description.

**Comment by HarryRybacki** [comment]
@GaryKriebel @brandonPurvis These PR titles can be edited after they have been opened. +1 to changing it to better reflect the purpose.

**Comment by samanehsan** [comment]
I just checked out the code and it looks good! First pass complete :octocat:

**Comment by samanehsan** [comment]
@sloria, I think this is good to go!

File: tests/test_elastic.py [hunk 1]
```diff
@@ -438,6 +441,40 @@ def test_search_partial(self):
         assert_equal(len(contribs['users']), 0)
 
 
+    def test_search_fullname_special_character(self):
+        """Verify that searching for a fullname with a special character yields
```
Comment (sloria) [inline tests/test_elastic.py]: Thank you for adding these tests. 

As per https://cosdev.readthedocs.org/en/latest/process/testing.html#general-testing-guidelines, we prefer descriptive test method names to docstrings, in order to make it easier to locate tests when they fail. I will convert these docstrings to code comments when I merge.

**Pull request merged**